modules/fb_instant_articles_api_rules/fb_instant_articles_api_rules.info

```diff
@@ -0,0 +1,5 @@
+name = Facebook Instant Articles API Rules
+description = Provide Drupal rules to manage facebook instant article using API.
+package = Facebook Instant Articles
+core = 7.x
+dependencies[] = fbia
```

Comment:
This dependency should be fb_instant_articles_api and it also depends on the Rules module.